oculus_ros2/src/sonar_viewer.cpp

```diff
@@ -40,7 +40,7 @@ SonarViewer::~SonarViewer() {}
 
 void SonarViewer::publishFan(const oculus_interfaces::msg::Ping& ros_ping_msg) const {
   // const int offset = ping->ping_data_offset(); // TODO(hugoyvrn)
-  const int offset = 229;  // quick fix TODO(hugoyvrn, why 229?)
+  const int offset = -16;  // quick fix TODO(hugoyvrn, why 229?)
```

Comment:
Why -16 ?

Reply:
For the same reason as 229: it works. In fact, this function is only called when we publish an image from a ros topic (from a rosbag, for example). In nominal real-time use, the offset is handled, but the information is not directly available in the ros topic. Thus, until a way of calculating the offset is found, an arbitrary value is used for rosbags.